libsolidity/interface/AssemblyStack.h

```diff
@@ -60,6 +60,10 @@ class AssemblyStack
 	/// Multiple calls overwrite the previous state.
 	bool parseAndAnalyze(std::string const& _sourceName, std::string const& _source);
 
+	/// Runs analysis step on the supplied block, returns false if input cannot be assembled.
+	/// Multiple calls overwrite the previous state.
+	bool analyze(assembly::Block const& _block, Scanner const* _scanner = nullptr);
```

Comment:
As far as I can see, the assembly stack does not provide a way to retrieve the result of the parsing step, so this function here alone is not really useful, is it?